app/views/docs/admin.phtml

```diff
@@ -1,7 +1,9 @@
-<p>The Appwrite API has two different modes it can use, the first mode is the client mode, which is the <b>default</b> mode, and the second one is the <b>admin</b> or server mode.</p>
+<p>You can use Appwrite to build both client and server integrations. Client integrations are for frontend and mobile applications, which use Appwrite's Client APIs and <a href="/docs/sdks#client">Client SDKs</a>. Server integrations are for backend and CLI applications, which use Appwrite's Server APIs and <a href="/docs/sdks#server">Server SDKs.</a></p>
```

Comment:
Should we also mention the Appwrite CLI here?

Reply:
Sure